product_docs/docs/pgd/5/parallelapply.mdx

```diff
@@ -5,16 +5,26 @@ navTitle: Parallel Apply
 
 ### What is Parallel Apply?
 
-Parallel Apply is a feature of PGD that allows a PGD node to use multiple writers per subscription. This generally increases the throughput of a subscription and improves replication performance.
+!!! Note Improved in PGD 5.2
+This feature has been enhanced in PGD 5.2 and is recommended for more scenarios. 
+!!!
 
-The transactional changes from the subscription are written by the multiple Parallel Apply writers. However, each writer ensures that the final commit of its transaction does not violate the commit order as executed on the origin node. If there is a violation, an error is generated and the transaction can be rolled back. 
+Parallel Apply is a feature of PGD that allows a PGD node to use multiple writers per subscription. This behavior generally increases the throughput of a subscription and improves replication performance.
 
-!!! Warning Possible deadlocks
-It may be possible that this out-of-order application of changes could trigger a deadlock. PGD currently resolves such deadlocks between Parallel Apply writers by aborting and retrying the transactions involved. If you experience a large number of such deadlocks, this is an indication that Parallel Apply is not a good fit for your workload and you should consider disabling it.
-!!!
+The transactional changes from the subscription are written by the multiple Parallel Apply writers. However, each writer ensures that the final commit of its transaction doesn't violate the commit order as executed on the origin node. If there's a violation, an error is generated and the transaction can be rolled back. 
```

Comment:
are they "multiple parallel apply writers" or when there are multiple writers we call it Parallel Apply - meaning if the the number of writers is 1 then we're not using paralell apply but any scenario where it's greater than 1 we're applying writes in parallel which we refer to as Parallel Apply - and when there's more than one writer there's these extra things we take into account

Reply:
```suggestion
When Parallel Apply is operating, the transactional changes from the subscription are written by multiple writers. However, each writer ensures that the final commit of its transaction doesn't violate the commit order as executed on the origin node. If there's a violation, an error is generated and the transaction can be rolled back. 
```